### PR TITLE
Only display answers after both players have guessed

### LINKED ISSUE #1: Do not display answers before all the players have guessed the capital
The game play program is not great because it  displays the answer to the guesses before all the players give their answers.

### PR DESCRIPTION
**Pull request by SoniaBarasa**
This pull request updates how answers are displayed in the game. Previously, answers were displayed after each guess, but now we wait until both players have guessed before displaying the answer.

* Resolves #1

### CODE REVIEW

**Comment by ahernank** [comment]
Great! So, before merging your pull request, you usually want a second pair of eyes in your code. You can add reviewers, using `Reviewers` above.

**Comment by ahernank** [comment]
For the title of this PR, similarly to the titles in issues, you can stick to words that describe (specifically) what this PR does. For example, `Update how answers are displayed in the game` or  even try to come up with a more specific title.

In the description of the PR, you might want to add more details on how your code solves this issue. 

Also, in the description is a good idea to use the first line to reference other issues. In this case, if issue #1 is fully solved by this PR, you can use `Resolves #1`. This will close issue #1 when this PR is merged. If the issue is not fully solved, you can use `Partially addresses #1`. This will close create a reference in issue #1, which is useful while working through bigger issues.

**Comment by ahernank** [comment]
> Great! So, before merging your pull request, you usually want a second pair of eyes in your code. You can add reviewers, using Reviewers above.

You might need to add us as collaborators first. Go to settings -> Colaborators -> Manage access; and you can search for Alistair and myself, using our GitHub usernames.

**Comment by ahernank** [comment]
In the description to link issues you need to use the `#` character, so to link issue 1, you would use #1. If you click #1, it will take you to the issue you are referencing. 

You can update your description text on this PR to make sure the issue is correctly referenced, which will ensure that when you merge this PR, the tag `Resolves` will close the issue you referenced.

File: gameofgame.py [hunk 1]
```diff
@@ -20,39 +20,50 @@
...
+    
 
 # Function to check if the guess is correct
 def check_guess(guess, country):
     capital = countries_capitals[country]
     if guess.lower() == capital.lower():
         return True
     else:
-        return False
+       return False
```
Comment (alimanfoo) [inline gameofgame.py:31]: Looks like there is inconsistent indentation here.

File: gameofgame.py [hunk 2]
```diff
@@ -20,39 +20,50 @@
...
     print("Player 1's turn")
-    guess = input(f"What is the capital of {country}? ")
-    if check_guess(guess, country):
-        print("Correct!")
-    else:
-        print("Incorrect.")
+    player1_guess = input(f"What is the capital of {country}? ")
+    
+    ##player2
+    print ("player 2 's turn")
```
Comment (alimanfoo) [inline gameofgame.py:44]: Here the formatting of the message text is different from player 1. There is no capitalisation, and there is a space before the apostrophe.

File: gameofgame.py [hunk 3]
```diff
@@ -20,39 +20,50 @@
...
+    capital = countries_capitals[country]
+    print (f"The correct answer is {capital}.")
+    
 
+     ##check the guesses and print results
+    if check_guess(player1_guess, country) and check_guess(player2_guess, country):
+        print("Both players guessed correctly!")
+    elif check_guess(player1_guess, country):
+        print("Player 1 guessed correctly!")
+        print("well done, you know your Geography well!")
```
Comment (alimanfoo) [inline gameofgame.py:57]: No capitalisation of the message text.

File: gameofgame.py [hunk 4]
```diff
@@ -20,39 +20,50 @@
...
+    elif check_guess(player2_guess, country):
+        print("Player 2 guessed correctly!")
```
Comment (alimanfoo) [inline gameofgame.py:59]: Player 2 will never get congratulated, that seems sad for player 2!

**Review by alimanfoo — COMMENTED** [review]
Hi @SoniaBarasa, this looks great :)

Just some small nits here, otherwise looks good.

**Review by SoniaBarasa — COMMENTED** [review]
@alimanfoo Thank you so much for the review, I have made made the changes as suggested.

File: gameofgame.py (hunk 1 above)
Comment (alimanfoo) [reply gameofgame.py:31]: OK, so now you have the same indentation on both these lines, but it is inconsistent with the rest of the file. All indentation in a Python module needs to be a multiple of 4 spaces.

File: gameofgame.py (hunk 3 above)
Comment (alimanfoo) [reply gameofgame.py:57]: Still no capitalisation of the message text here.

File: gameofgame.py (hunk 4 above)
Comment (alimanfoo) [reply gameofgame.py:59]: Thanks for adding the message :)

File: gameofgame.py [hunk 5]
```diff
@@ -20,39 +20,52 @@
...
+        print("well done, you know your Geography well!")
+    else:
+        print("Neither player guessed correctly.")
+        print("You both should learn about capital cities in the world!")
```
Comment (alimanfoo) [inline gameofgame.py:63]: Haha, I like this :)

**Comment by SoniaBarasa** [comment]
Hello @ahernank @alimanfoo , thank you very much for all your feedback. I have implemented all the suggestions now and the code looks good. I am thus going to close this issue. Cheers!

**Pull request merged**